guides/source/testing.md

```diff
@@ -481,20 +481,22 @@ In the `test_should_get_index` test, Rails simulates a request on the action cal
 The `get` method kicks off the web request and populates the results into the response. It accepts 4 arguments:
 
 * The action of the controller you are requesting. This can be in the form of a string or a symbol.
-* An optional hash of request parameters to pass into the action (eg. query string parameters or article variables).
-* An optional hash of session variables to pass along with the request.
-* An optional hash of flash values.
+* `params:` option with a hash of request parameters to pass into the action (eg. query string parameters or article variables).
```

Comment:
Missing dot after the "e" of "e.g.". Also could you please wrap your lines around 80 chars editing the guides ? That would be awesome!